### ISSUE TITLE
Scoreboard overlaps with courses

### ISSUE THREAD

**Issue by jrodal98** [SyllabiShare/syllabi-share#4]
I imagine there will be UI changes eventually and this issue isn't really that big of a deal, since it doesn't affect the functioning of the app.

The scoreboard overlaps with the courses if the window is small enough, such as when my browser is taking up half of my laptop screen
![window_bug](https://user-images.githubusercontent.com/35352333/82721791-ab36b600-9c8e-11ea-8741-36e52ebdb4d9.gif)

**Comment by verndrade** [comment]
Yeah thats a problem with using a fixed div. Will try to make the UI more responsive after more necessary functionality is achieved

**Comment by verndrade** [comment]
leaderboard is now hidden but will be fixed before made visible again
